src/control/lib/helper/app.go

```diff
@@ -0,0 +1,246 @@
+//
+// (C) Copyright 2020 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package helper
```

Comment:
I know we talked about this, but I don't remember the motivation behind putting this stuff into a new package instead of using the `pbin` package.

Reply:
I think we were courting the idea of whether we'd ever want non-privileged helper applications. Doesn't seem probable in my view, though. I'd be okay with moving it to pbin.

Reply:
Ah, right. Yeah, after seeing the code, `pbin` seems like the right home for it, given how many imports there are from that package anyhow.